### PR TITLE
Add molecule tests to frr component

### PR DESCRIPTION
**Pull request by fernandoroyosanchez**
Add molecule tests to frr component

### CODE REVIEW

File: roles/edpm_frr/molecule/default/verify.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    test_helper_dir: "../../../../molecule/test-helpers"
+  tasks:
+    - name: ensure expected directories exist
+      ansible.builtin.include_tasks: "{{test_helper_dir}}/verify_dir.yaml"
+      loop:
+        - "/var/lib/edpm-config/container-startup-config/frr"
+        - "/var/lib/config-data/ansible-generated/frr"
+        - "/var/log/containers/frr"
+        - "/etc/tmpfiles.d/"
+        - "/var/lib/config-data/ansible-generated/frr/etc/frr/frr.conf"
```
Comment (luis5tb) [inline roles/edpm_frr/molecule/default/verify.yml:15]: this is not a directory

File: roles/edpm_frr/tasks/install.yml [hunk 2]
```diff
@@ -15,15 +15,19 @@
...
-    mode: "{{ item.mode }}"
+    setype: "{{ item.setype | default('container_file_t') }}"
+    owner: "{{ item.owner | default(ansible_user) }}"
+    group: "{{ item.group | default(ansible_user) }}"
+    mode: "{{ item.mode | default(omit) }}"
   loop:
     - {'path': /var/log/containers/frr, 'setype': container_file_t, 'mode': '0750'}
     - {'path': "{{ edpm_frr_config_basedir }}", 'setype': container_file_t, 'mode': '0750'}
     - {'path': /run/frr, 'setype': container_file_t, 'mode': '0755'}
+    - {"path": "/etc/tmpfiles.d/", "owner": "root", "group": "root"}
```
Comment (luis5tb) [inline roles/edpm_frr/tasks/install.yml:30]: do we need to mount this? is this being used for things not related to testing?

**Review by luis5tb — CHANGES_REQUESTED** [review]

File: roles/edpm_frr/tasks/install.yml [hunk 3]
```diff
@@ -15,17 +15,22 @@
...
 
 - name: Create persistent directories
+  become: true
   ansible.builtin.file:
     path: "{{ item.path }}"
     state: directory
-    setype: "{{ item.setype }}"
-    mode: "{{ item.mode }}"
+    setype: "{{ item.setype | default('container_file_t') }}"
+    owner: "{{ item.owner | default(ansible_user) }}"
```
Comment (luis5tb) [inline roles/edpm_frr/tasks/install.yml:23]: it seems ansbile_user is not defined

File: roles/edpm_frr/tasks/configure.yml [hunk 4]
```diff
@@ -51,7 +52,7 @@
   become: true
   ansible.builtin.template:
     src: daemons.j2
-    dest: "{{ edpm_frr_config_basedir }}/etc/frr/daemons"
```
Comment (luis5tb) [inline roles/edpm_frr/tasks/configure.yml:54]: I think this should still be /etc/frr/daemons and sema for frr.conf

**Comment by fernandoroyosanchez** [comment]
recheck

File: roles/edpm_frr/tasks/install.yml [hunk 5]
```diff
@@ -25,7 +26,20 @@
     - {'path': "{{ edpm_frr_config_basedir }}", 'setype': container_file_t, 'mode': '0750'}
     - {'path': /run/frr, 'setype': container_file_t, 'mode': '0755'}
 
+- name: Create persistent directories (just for testing purpose)
```
Comment (luis5tb) [inline roles/edpm_frr/tasks/install.yml:29]: if this is needed just for testing only, shouldn't it be on the prepare.yaml instead?

Comment (fernandoroyosanchez) [reply roles/edpm_frr/tasks/install.yml]: moved, make sense!

**Comment by fernandoroyosanchez** [comment]
recheck

**Comment by fernandoroyosanchez** [comment]
recheck

**Comment by fernandoroyosanchez** [comment]
recheck

**Pull request merged**